.env_example

```diff
@@ -0,0 +1,16 @@
+; Oauth parameters.
+OAUTH_CONSUMER_TOKEN =
+OAUTH_SECRET_TOKEN   =
+OAUTH_ENDPOINT       = 'https://meta.wikimedia.org/w/index.php?title=Special:OAuth'
+OAUTH_REDIR          = 'https://meta.wikimedia.org/wiki/Special:OAuth/authenticate?'
+OAUTH_CALLBACK       = 'oob'
```

Comment:
'Out Of Band', which isn't really correct because we do actually use a callback. (OOB usually means e.g. a desktop application is requesting the auth.) In our case, the callback is fixed by the consumer registration on Meta, so actually this will be ignored; it does have to be something though.